v2/professions.js

```diff
@@ -108,9 +108,39 @@
 			"specialization" : 12,
 			"skills" : [ /* ... */ ]
 		}
-	}
+	},
+	// List of all non-weapon skills that can be equipped by this
+	// profession (doesn't include transform skills and such).
+	"skills" : [
+		{
+			"id"   : 34,
+			"slot" : "Profession_1",
+			"type" : "Profession"
```

Comment:
I would add that as `"specialization": 40`, like it is currently done with weapons.